dnSpy/dnSpy/Search/FrameworkFileUtils.cs

```diff
@@ -470,7 +470,7 @@ static class FrameworkFileUtils {
 			}
 
 			// .NET Framework
-			if (GacInfo.IsGacPath(filename))
+			if (! string2.IsNullOrWhiteSpace(filename) && GacInfo.IsGacPath(filename))
```

Comment:
Without the space between the negation operator (`!`) and the call to `IsNullOrWhiteSpace` to match the rest of the dnSpy codebase.
```suggestion
			if (!string2.IsNullOrWhiteSpace(filename) && GacInfo.IsGacPath(filename))
```